Dockerfile.build

```diff
@@ -5,14 +5,15 @@ ARG BAZELISK_VERSION=1.8.0
 ARG PI_COMMIT=a5fd855d4b3293e23816ef6154e83dc6621aed6a
 ARG BMV2_COMMIT=b0fb01ecacbf3a7d7f0c01e2f149b0c6a957f779
 ARG JDK_URL=https://mirror.bazel.build/openjdk/azul-zulu11.29.3-ca-jdk11.0.2/zulu11.29.3-ca-jdk11.0.2-linux_x64.tar.gz
-ARG LLVM_REPO_NAME="deb http://apt.llvm.org/stretch/  llvm-toolchain-stretch-11 main"
+ARG LLVM_REPO_NAME="deb http://apt.llvm.org/buster/ llvm-toolchain-buster main"
```

Comment:
I create this a few weeks ago and I forgot why I switch from stretch to buster (might be due to the python3 version)